src/server/api/schema.js

```diff
@@ -649,23 +650,71 @@ const rootMutations = {
         return userData;
       }
     },
-    resetUserPassword: async (_, { organizationId, userId }, { user }) => {
+    resetUserPassword: async (
+      _,
+      { organizationId, userId },
+      { user, loaders }
+    ) => {
       if (user.id === userId) {
         throw new Error("You can't reset your own password.");
       }
+      // might need to impliment user password change available elsewhere.
```

Comment:
Can you expand more about this comment?

Reply:
Oh!  Currently an administrator is not allowed to change their own password.  Cody suggested that we should change that, so I made a note of it in the code.